frontend/package.json

```diff
@@ -59,10 +60,11 @@
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-i18next": "^14.0.5",
+    "react-idle-timer": "^5.7.2",
     "redis": "^4.6.13",
     "remix-toast": "^1.1.0",
     "source-map-support": "^0.5.21",
-    "styled-components": "^6.1.8",
+"styled-components": "^6.1.8",
```

Comment:
The formatting seems to be messed up here.. also do we need `styled-components`?

@sebastien-comeau any comment?

Reply:
Styled components should be removed, yes.